components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/APIDefinitionFromSwagger20.java

```diff
@@ -281,51 +281,71 @@ public String createAPIDefinition(API api) throws APIManagementException {
         String endpoints = environment.getApiGatewayEndpoint();
         String[] endpointsSet = endpoints.split(",");
         Set<URITemplate> uriTemplates = api.getUriTemplates();
+        Set scopes = api.getScopes();
 
         if (endpointsSet.length < 1) {
             throw new APIManagementException("Error in creating JSON representation of the API" + identifier.getApiName());
         }
 
-        JSONObject swaggerObject;
-
-        try {
-            swaggerObject = (JSONObject) parser.parse(swaggerObjectTemplate);
-
-            //Create info object
-            JSONObject infoObject = (JSONObject) parser.parse(infoObjectTemplate);
-            infoObject.put("title", api.getId().getApiName());
-            infoObject.put("description", api.getDescription());
-
-            //Create contact object and map business owner info
-            JSONObject contactObject = (JSONObject) parser.parse(contactObjectTemplate);
-            contactObject.put("name", api.getBusinessOwner());
-            contactObject.put("email", api.getBusinessOwnerEmail());
-            //put contact object to info object
-            infoObject.put("contact", contactObject);
-
-            //Create licence object
-            JSONObject licenceObject = (JSONObject) parser.parse(licenceObjectTemplate);
-
-            infoObject.put("licence", licenceObject);
-            infoObject.put("version", api.getId().getVersion());
-
-            //add info object to swaggerObject
-            swaggerObject.put("info", infoObject);
-
-            for (URITemplate uriTemplate : uriTemplates) {
+        JSONObject swaggerObject = new JSONObject();
+
+        //Create info object
+        JSONObject infoObject = new JSONObject();
+        infoObject.put("title", api.getId().getApiName());
+        infoObject.put("description", api.getDescription());
+
+        //Create contact object and map business owner info
+        JSONObject contactObject = new JSONObject();
+        contactObject.put("name", api.getBusinessOwner());
+        contactObject.put("email", api.getBusinessOwnerEmail());
+        //put contact object to info object
+        infoObject.put("contact", contactObject);
+
+        //Create licence object
+        JSONObject licenceObject = new JSONObject();
+
+        infoObject.put("licence", licenceObject);
+        infoObject.put("version", api.getId().getVersion());
```

Comment:
Resolved in the commit [1]
[1] - https://github.com/wso2/carbon-apimgt/commit/8c284e54fda30c37591be7207fea99b252143945